### DISCUSSION TITLE
remove cut/copy/paste modal in iphone

### DISCUSSION THREAD

**Discussion by joshua-watersport** [ritikbanger/react18-input-otp#20] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/121116326/228993502-4a101beb-e746-432f-8b45-2c55196d00ee.png)

when copy the otp on the ios device, I get the issue like a screenshot.

How can I fix?

**Comment by ritikbanger** [comment]
Do you know if this is breaking any functionality?

From the screenshot, I recommend you trigger your auth API as soon as the length of the otp===4, alternatively, you can unfocus the input.

**Reply by joshua-watersport** [reply]
How can I unfocus the input for react18-input-otp?

**Reply by ritikbanger** [reply, marked as answer]
Use the blur() method to remove the focus from an element
